include/ibamr/ibamr_enums.h

```diff
@@ -213,6 +213,38 @@ enum_to_string<LevelSetOrder>(LevelSetOrder val)
     return "UNKNOWN_LEVEL_SET_ORDER";
 } // enum_to_string
 
+/*!
+ * \brief Enumerated type for different level set time stepping schemes
+ */
+enum LevelSetTimeStepping
+{
+    GAUSS_SEIDEL_PSEUDO_TS = 1,
+    RK1_TS = 2,
+    RK2_TS = 3,
+    UNKNOWN_LEVEL_SET_TS = -1
+}; // LevelSetTimeStepping
+
+template <>
+inline LevelSetTimeStepping
+string_to_enum<LevelSetTimeStepping>(const std::string& val)
+{
+    if (strcasecmp(val.c_str(), "GAUSS_SEIDEL") == 0) return GAUSS_SEIDEL_PSEUDO_TS;
+    if (strcasecmp(val.c_str(), "RK1") == 0) return RK1_TS;
```

Comment:
If RK1 is just forward Euler --- can we just list forward Euler?

Reply:
I thought of that, but then I thought to be consistent with other RK options it is better to write it as RK1. I don't have a strong opinion and can change it.